components/form/__tests__/index.test.tsx

```diff
@@ -778,7 +800,7 @@ describe('Form', () => {
     );
 
     // if getByLabelText can get element, then it is a validate field with form control and label
-    expect(screen.queryByLabelText('test')).not.toBeInTheDocument();
+    expect(screen.queryByLabelText('test')).toBeInTheDocument();
```

Comment:
Why change this?

Reply:
The Form.Item has an prop label 'test', the input element has an aria-label attribute also set to 'test' and it can locate based on the input's aria-label attribute by screen.queryByLabelText('test').

Reply:
@zombieJ Please help to check this.